docs/runbooks/HPPSharingPoolPathWithOS.md

```diff
@@ -0,0 +1,30 @@
+# HPPSharingPoolPathWithOS
+
+## Meaning
+
+The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
+If HPP is sharing a filesystem with other critical components (kubelet / OS) its possible that HPP PVs will cause node disk pressure.
+## Impact
+
+Possible degraded node operation.
+
+## Diagnosis
+
+- Check which HPP pool is being shared with OS by looking at DaemonSet logs
```

Comment:
Either ad a dot at the end or remove the dots from the other steps.

Reply:
Done